man/man5/zfs-module-parameters.5

```diff
@@ -2139,6 +2139,17 @@ resilvers per leaf device, given in bytes.
 Default value: \fB41943040\fR.
 .RE
 
+.sp
+.ne 2
+.na
+\fBzfs_scan_disable_progress\fR (int)
+.ad
+.RS 12n
+Set to disable scanning progress. This can be used for both scrubs and resilvers.
+.sp
+Default value: \fB0\fR.
```

Comment:
Can we structure this more like `zfs_remove_max_bytes_pause` which does something very similar, and call it `zfs_scan_max_bytes_pause` for consistency.  I notice `zfs_remove_max_bytes_pause` does not appear in the man page and we could omit this one as well since it's only for the test suite.

Reply:
You're right, `zfs_remove_max_bytes_pause` is going to be problematic on a 32-bit system.  OK, then let's convert it to a flag as well since we don't use the functionality to pause at a specific byte count. How about `zfs_remove_pause` and `zfs_scan_pause`?